apps/daimo-mobile/src/view/TabNav.tsx

```diff
@@ -158,6 +159,7 @@ function HomeTab() {
         <HomeStack.Screen name="Home" component={HomeScreen} />
         <HomeStack.Screen name="QR" component={QRScreen} />
         <HomeStack.Screen name="HistoryOp" component={HistoryOpScreen} />
+        <HomeStack.Screen name="Account" component={AccountScreen} />
```

Comment:
also nit but might call this "profile"?

Reply:
matched the figma

you can end up on this page for any ethereum account:  daimo, ens, bare address.

Reply:
not yet done: avoid linking to account page for accounts we know you shouldn't send to directly, like `payment link`.

+ relatedly, on a transaction page claiming a payment link, we should link to the original sender of the link.

Reply:
done, see below